### PR TITLE
test: ssh_is_opened keeps on waiting in case of exceptions.

### PR DESCRIPTION
**Pull request by merlin-northern**
Changelog: None
Signed-off-by: Peter Grzybowski <peter@northern.tech>

### CODE REVIEW

File: testutils/infra/device.py [hunk 1]
```diff
@@ -110,7 +110,20 @@ def ssh_is_opened(self, wait=60 * 60):
         Keyword arguments:
         wait - Timeout (in seconds)
         """
-        self.run("true", hide=True, wait=wait)
+        waited = -1
```
Comment (oleorhagen) [inline testutils/infra/device.py:113]: How about we use the `redo` package here as well?

https://github.com/mendersoftware/integration/blob/master/testutils/infra/container_manager/docker_compose_base_manager.py#L21

**Review by oleorhagen — COMMENTED** [review]
Nice find!

**Comment by merlin-northern** [comment]
> @merlin-northern @oleorhagen What's the status here?

the status: this is best-effort and "as soon as we have a spare moment" thing. I have not looked into redo package, so it waits for my next QA duty (at most, perhaps will find time to do it sonner)

**Comment by merlin-northern** [comment]
> > @merlin-northern @oleorhagen What's the status here?
> 
> the status: this is best-effort and "as soon as we have a spare moment" thing. I have not looked into redo package, so it waits for my next QA duty (at most, perhaps will find time to do it sonner)

I still remember about it, got delayed a bit during my BM but will try today if time permits to take a look.

File: testutils/infra/device.py (hunk 1 above)
Comment (merlin-northern) [reply testutils/infra/device.py:114]: done.

**Comment by merlin-northern** [comment]
@mender-test-bot start pipeline please

**Comment by merlin-northern** [comment]
@alfrunes only now I managed to get to it.

**Review by tranchitella — APPROVED** [review]

**Pull request merged**